### PR TITLE
Fixed a few JavaDoc twerks

### PR DESCRIPTION
**Pull request by payne911**
The first two commits are irrelevant. (The second one reverts the first one.)

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/math/Interpolation.java [hunk 1]
```diff
@@ -294,14 +294,14 @@ public Bounce (int bounces) {
 		}
 
 		private float out (float a) {
```
Comment (crykn) [inline gdx/src/com/badlogic/gdx/math/Interpolation.java:296]: ^You forgot to change this parameter.

**Comment by Darkyenus** [comment]
I'm not a fan of these changes. I understand the motivation, but a *normalized value* is not really a thing (as far as I know) and I have never encountered this naming in the context of interpolation. `progress` (as used in `lerp()`) seems to fit better.

File: gdx/src/com/badlogic/gdx/math/Interpolation.java (hunk 1 above)
Comment (payne911) [reply gdx/src/com/badlogic/gdx/math/Interpolation.java]: Thanks, it's fixed!

I was meant to forget one since I've done those edits in GitHub directly.

**Comment by payne911** [comment]
> I'm not a fan of these changes. I understand the motivation, but a _normalized value_ is not really a thing (as far as I know) and I have never encountered this naming in the context of interpolation. `progress` (as used in `lerp()`) seems to fit better.

This is where I took the inspiration from:

image

`progress` does not convey the idea that the input value should be clamped between 0 and 1, as is a requirement mentioned by the documentation:

image

The word `normalized`, however, does have a mathematical definition which recalls a lot what the input requires to be. No one there, for example, seems confused by the use of the word: https://stats.stackexchange.com/questions/70801/how-to-normalize-data-to-0-1-range

Many other suggestions over there: https://english.stackexchange.com/questions/275734/a-word-for-a-value-between-0-and-1-inclusive

**Comment by NathanSweet** [comment]
Normalized may be slightly more descriptive (though the range to which you normalize something does not need to be 0-1) but I don't think it's very important here, since before and after the PR the javadoc still needs to say the range is 0 to 1 (funny enough the value can actually be outside the 0-1 range, if the results are what you want). Changing the variable names from `a` to `n` doesn't really make anything more clear, need better reasons to make changes. I'll merge the javadoc typo separately.

**Comment by payne911** [comment]
> Normalized may be slightly more descriptive (though the range to which you normalize something does not need to be 0-1) but I don't think it's very important here, since before and after the PR the javadoc still needs to say the range is 0 to 1 (funny enough the value can actually be outside the 0-1 range, if the results are what you want). Changing the variable names from `a` to `n` doesn't really make anything more clear, need better reasons to make changes. I'll merge the javadoc typo separately.

Sure, but `alpha` is just plainly **wrong** as a parameter name. `norm` at least reuses a word employed elsewhere in the libGDX API (namely: `MathUtils#norm`) with a clear definition of the goal. `alpha` is generally understood by *any* programmer as relating to transparency, and that is also how that word is used throughout the libGDX API, *except* here.

The `n` thing is purely to reproduce what was being done with the reduce version one-letter-version of the primary parameter name (`alpha` -> `a`, thus `norm` -> `n`). It's a secondary aspect of the real goal of this PR.

**Comment by Darkyenus** [comment]
Well, alpha can't be plainly wrong, because it does not actually mean anything, it is just a letter. It could just as well be called `x` or something. Equating alpha to transparency is so niche, that it does not even appear in [Alpha (disambiguation)](https://en.wikipedia.org/wiki/Alpha_(disambiguation)) page on Wikipedia.

**Pull request closed without merging**